pkg/controllers/securitypolicy/securitypolicy_controller.go

```diff
@@ -253,13 +253,13 @@ func (r *SecurityPolicyReconciler) setSecurityPolicyReadyStatusTrue(ctx context.
 func (r *SecurityPolicyReconciler) setSecurityPolicyReadyStatusFalse(ctx context.Context, secPolicy *v1alpha1.SecurityPolicy, transitionTime metav1.Time, err *error) {
 	newConditions := []v1alpha1.Condition{
 		{
-			Type:    v1alpha1.Ready,
-			Status:  v1.ConditionFalse,
-			Message: "NSX Security Policy could not be created/updated",
-			Reason: fmt.Sprintf(
+			Type:   v1alpha1.Ready,
```

Comment:
https://github.com/vmware-tanzu/nsx-operator/pull/791/files#diff-f45f721ba06ab477de36df3522bb24718a5f69d5747f3d6b5ae543fe98d68877L245
Seemly, as the new convention to update status reason/message, we should also update the StatusReady True condition as well

Reply:
This is also apply to other controllers.